service/src/main/resources/changelog/changesets/20230124_fetch_sra_to_bam_io_updates.yaml

```
@@ -0,0 +1,152 @@
+databaseChangeLog:
+  - changeSet:
+      id: "update fetch_sra_to_bam inputs/outputs to match the target workspace for public preview"
+      author: mspector
+      changes:
+        - update:
+            tableName: run_set
+            where: run_set_id='10000000-0000-0000-0000-000000000008'
+            columns:
+              - column:
+                  name: input_definition
+                  value: |
+                    [
+                      {
+                        "input_name": "fetch_sra_to_bam.Fetch_SRA_to_BAM.SRA_ID",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "sra_id"
+                        }
+                      },
+                      {
+                        "input_name": "fetch_sra_to_bam.Fetch_SRA_to_BAM.docker",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      },
+                      {
+                        "input_name": "fetch_sra_to_bam.Fetch_SRA_to_BAM.machine_mem_gb",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "Int"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
+                        }
+                      }
+                    ]
+              - column:
+                  name: output_definition
+                  value: |
+                    [
+                      {
+                        "output_name": "fetch_sra_to_bam.sra_metadata",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "sra_metadata"
```

These also need updating to the new outputs style: 
```suggestion
                        "destination": {
                          "type": "record_update",
                          "record_attribute": "sra_metadata"
                        }
```